CONFIGDOC.md

```diff
@@ -60,7 +60,7 @@ The **Run MATLAB Tests** build step enables you to generate different types of t
 
 If you do not select any of the test artifact check boxes, the tests still run, and test failures fail the build.
 
-The **Run MATLAB Tests** build step produces a MATLAB script file named `runMatlabTests.m` in the project workspace. The plugin uses this file to run the tests and generate the test artifacts. You can review the contents of the script to understand the testing workflow.
+The **Run MATLAB Tests** build step produces a MATLAB script file, Content of this file will be printed on console log during build. The plugin uses this file to run the tests and generate the test artifacts. You can review the contents of the script to understand the testing workflow.

```

Comment:
Thanks, Nikhil. How about some additional changes?

The **Run MATLAB Tests** build step produces a MATLAB script file and uses it to run the tests and generate the test artifacts. The plugin writes the contents of this file to the build log. You can review the build log in **Console Output** to understand the testing workflow.

Reply:
Done.